src/components/Limit.vue

```diff
@@ -3,7 +3,7 @@
 		<input
 			class="querybuilder__limit-checkbox"
 			type="checkbox"
-			name="limit"
+			id="limit"
```

Comment:
This id also would go away once it's decided how the label will be handled. This will be replaced with the checkbox component anyway

Reply:
This change to that attribute should be in the previous PR #119